beets/importer.py

```diff
@@ -521,17 +521,18 @@ def skip(self):
 
     # Convenient data.
 
-    def chosen_ident(self):
-        """Returns identifying metadata about the current choice. For
-        albums, this is an (artist, album) pair. For items, this is
-        (artist, title). May only be called when the choice flag is ASIS
-        or RETAG (in which case the data comes from the files' current
-        metadata) or APPLY (data comes from the choice).
+    def chosen_info(self):
+        """Return a dictionary of metadata about the current choice.
+        May only be called when the choice flag is ASIS or RETAG
+        (in which case the data comes from the files' current metadata)
+        or APPLY (in which case the data comes from the choice).
         """
+        assert(self.choice_flag in (action.ASIS, action.RETAG, action.APPLY))
```

Comment:
I don't think the parentheses are necessary:
```suggestion
        assert self.choice_flag in (action.ASIS, action.RETAG, action.APPLY)
```

Reply:
In fact, another (perhaps) way to catch this is to just put `assert False` at the end, after the `elif`.